Models/Administrative.cs

```diff
@@ -0,0 +1,28 @@
+using System;
+
+namespace staff
+{
+    class Administrative: Staff
+    {
+         
+        public string AdminNo    
+        {get;set;}
```

Comment:
write get, set accessors on the same line of property for the auto implementation case